Tests/ML/test_metrics.py

```diff
@@ -33,21 +33,23 @@ def test_calculate_dice1() -> None:
     g1 = "g1"
     zero = np.zeros((3, 3, 3))
     one = np.ones((3, 3, 3))
+    missing_labels = [False]
 
     # ground truth is expected in one-hot encoding, but the segmentation is a map with class indices in each voxel
-    def assert_metrics(segmentation: np.ndarray, ground_truth: np.ndarray, expected_dice: float) -> None:
-        a = metrics.calculate_metrics_per_class(segmentation, ground_truth,
+    def assert_metrics(segmentation: np.ndarray, ground_truth: np.ndarray, missing_label: List[bool],
+                       expected_dice: float) -> None:
+        a = metrics.calculate_metrics_per_class(segmentation, ground_truth, missing_label,
```

Comment:
If you follow my suggestion to add "missing_labels" as an optional field, and recognize "None" as "nothing is missing", you can greatly simplify also all these tests here.